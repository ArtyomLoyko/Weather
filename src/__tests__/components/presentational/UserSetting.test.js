import React from 'react';
import { shallow } from 'enzyme';
import UserSetting from './../../../components/presentational/UserSetting';

describe('UserSetting', () => {
    const userSetting = shallow(<UserSetting />);

    it('renders properly', () => {
        expect(userSetting).toMatchSnapshot();
    });
});
