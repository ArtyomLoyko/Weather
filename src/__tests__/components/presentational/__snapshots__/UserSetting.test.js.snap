// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserSetting renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <UserSetting />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          User Setting
</h2>,
        <withRouter(RegistrationFormContainer) />,
      ],
      "className": "user-setting",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "User Setting",
        },
        "ref": null,
        "rendered": "User Setting",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            User Setting
</h2>,
          <withRouter(RegistrationFormContainer) />,
        ],
        "className": "user-setting",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "User Setting",
          },
          "ref": null,
          "rendered": "User Setting",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
